src/pages/RecipePage/RecipePage.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from "react";
+
+import axios from "axios";
+import parse from "html-react-parser";
+import { Link } from "react-router-dom";
+
+import styles from "./RecipePage.module.scss";
+
+export const RecipePage = () => {
+  const [recipeFetched, setRecipe] = useState([]);
```

Comment:
тут должны быть указаны типы, тогда ниже не надо будет any использовать 
```suggestion
  const [recipeFetched, setRecipe] = useState<YourType[]>([]);
```

Reply:
Использовать `any` в каких-то супер крайних случаях, тут такого нет